pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -241,7 +241,11 @@ func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *opv1al
 	}
 
 	// Fill all defaults explicitly
-	network.FillDefaults(&networkAddonsConfig.Spec, prev)
+	if err := network.FillDefaults(&networkAddonsConfig.Spec, prev); err != nil {
+		log.Printf("failed to fill defaults: %v", err)
+		errors.Wrapf(err, "failed to fill defaults: %v", err)
```

Comment:
looks like a bug, we don't use the return value. i don't think that err is updated in place

Reply:
Yes you are right, Can I create a new PR to fix this?